backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/SysMLElementSerializer.java

```diff
@@ -124,7 +147,169 @@ public String caseDefinition(Definition def) {
 
         return builder.toString();
     }
-    
+
+    public String caseLiteralExpression(LiteralExpression expression) {
```

Comment:
Please split this in several caseXXLiteralExpression.

Be carefull you forgot to add the @Override annotation.

Reply:
done

Reply:
You still forgot the @Override annotation